makeConfig.js

```diff
@@ -0,0 +1,70 @@
+import globals from 'globals';
+import eslintConfigPrettier from 'eslint-config-prettier';
+import typescriptEslintPlugin from 'typescript-eslint';
+
+import arrayFuncConfig from './configs/array-func.js';
+import eslintConfig from './configs/eslint.js';
+import eslintCommentsConfig from './configs/eslint-comments.js';
+import functionalConfig from './configs/functional.js';
+import importConfig from './configs/import.js';
+import jestConfig from './configs/jest.js';
+import simpleImportSortConfig from './configs/simple-import-sort.js';
+import stylisticConfig from './configs/stylistic.js';
+import unicornConfig from './configs/unicorn.js';
+import typescriptConfig from './configs/typescript.js';
+import vitestConfig from './configs/vitest.js';
+
+const configs = [
+  arrayFuncConfig,
+  eslintCommentsConfig,
+  eslintConfig,
+  functionalConfig,
+  importConfig,
+  simpleImportSortConfig,
+  stylisticConfig,
+  typescriptConfig,
+  unicornConfig,
+];
+
+const makeOptionsConfig = (additionalGlobals = {}) => ({
```

Comment:
nit: `additionalGlobals` seems to never be used anywhere

Reply:
Yes!  An earlier version used it but it should be included automatically as part of the plugins so then I removed it.